tests/remote_config/rc_expected_requests_asm_features.json

```diff
@@ -428,7 +438,30 @@
 						"hash": "9221dfd9f6084151313e3e4920121ae843614c328e4630ea371ba66e2f15a0a6"
 					}
 				]
-			},
+			}
+		]
+	},
+	{
+		"client": {
+			"state": {
+				"root_version": 1,
+				"targets_version": 11,
```

Comment:
Shouldn't this be `12`? As `11` was the previous one? cc: @cbeauchesne

Reply:
Not in this case, there are failure cases inside the tests as well where tracers should outright reject the payload and keep the previous state.

That being said, what's tricky here is that we had a bug where the last test was not being generated and evaluated by system tests. So this PR doesn't actually add the "hardcoded config name" test but instead adds the last test (which was something that should fail in the tracers) that has been in the script but never evaluated.

A later PR that @coignetp made fixes this bug and generates the final expected request for the hardcoded config case which should succeed and will expect the tracers to bump the targets version.